style.css

```diff
@@ -0,0 +1,665 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e5e5e5;
+  font-family: "Poppins", sans-serif;
+}
+
+.about-me {
+  font-weight: 700;
+  font-family: "Poppins", sans-serif;
+  font-size: 50px;
+  margin: 2px;
+  align-items: center;
+  color: hsl(218, 54%, 20%);
+}
+
+.AboutContent {
+  padding-left: 1px;
+  font-weight: 400;
+  font-size: 17px;
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+}
+
+/* Form styles */
+.contact-application {
+  display: flex;
+  padding-left: 100px;
+  padding-top: 40px;
+  align-items: center;
+  font-family: "Poppins", sans-serif;
+  justify-content: center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-position: center center;
+  background-color: hsla(234, 100%, 69%, 1);
+  background-image: url("Asset/contact-bg-d.png");
+  border-radius: 60px 0 0 0;
+}
```

Comment:
- [ ] As mentioned by the previous reviewer It's a good practice to not repeat CSS properties that you'll use multiple times, (DRY). It seems like you're using the `display: flex;` property numerous times and it's best to write it in its own class like the one below and use that class name where ever you need it in your HTML.
```
    .flex-row {
      display: flex;
    }
    .flex-column {
      display: flex;
      flex-direction: column;
    }
```
- [ ] Here's a screenshot of how many times `display: flex;` has been used (22 times):
  ![Code_6Br5PGXArx](https://github.com/melkamu12/Portfolio-DesktopVersionM/assets/104968236/8ab253ec-f4ac-4a7f-880f-067224215b02)